src/main/java/com/conveyal/datatools/manager/controllers/api/ServerController.java

```diff
@@ -326,7 +328,12 @@ private static void verifyS3WritePermissions(OtpServer server, Request req) {
 //        }
     }
 
-    /** Validate that AMI exists and value is not empty. */
+    /**
+     * Validate that AMI exists and value is not empty.
+     *
+     * TODO: Should we warn user if the AMI provided is older than the default AMI registered with this application as
+     *   DEFAULT_AMI_ID?
```

Comment:
Might be worthwhile to do this in case there are any security problems that arise from using out-of-date UIs.